Wrappers/Python/test/test_algorithms.py

```diff
@@ -378,15 +380,17 @@ def test_FISTA_Denoising(self):
         print ("FISTA Denoising Poisson Noise Tikhonov")
         # adapted from demo FISTA_Tikhonov_Poisson_Denoising.py in CIL-Demos repository
         #loader = TestData(data_dir=os.path.join(sys.prefix, 'share','ccpi'))
-        loader = TestData()
-        data = loader.load(TestData.SHAPES)
+        # loader = TestData()
+        # data = loader.load(TestData.SHAPES)
```

Comment:
commented old code - there's quite a lot throughout this file but I won't mark them all individually.